### DISCUSSION TITLE
https://plugin-components.vuejs.press/zh/guide/notice.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#633] [Announcements]
# https://plugin-components.vuejs.press/zh/guide/notice.html

该插件提供了一个全局组件 Notice 供你向访问者显示一些通知。 你可以通过公告配置中的“通知”选项进行配置。 用法 你可以为站点的不同路径设置多个公告，为此，请设置插件选项的 rootComponents.notice 为你所需要的公告配置数组。 每个公告配置需要包含一个 path 或 match 选项，用于匹配路径。path 选项为字符串，匹配所...

https://plugin-components.vuejs.press/zh/guide/notice.html

**Comment by AndersonHJB** [comment]
通知位置可以调整的话，就更好了
